src/about/events/bct12020.md

```diff
@@ -253,7 +253,7 @@ robust and easy to use.
 
```

Comment:
I actually didn't bother with the event-pages as those are temporary, and might include older products and code.

Reply:
So it's fine for now, but for the other submissions perhaps we don't need to add poplinks to eventpages